roles/telemetry_autoscaling/tasks/verify_autoscaling.yml

```
@@ -40,3 +40,12 @@
     {{ openstack_cmd }} metric status --fit-width;
   register: result
   failed_when: result.rc >= 1
+
+# This works when using prometheus as the metrics backend
+- name: Verify that PROMETHEUS is enabled in RHOSO18
```

```suggestion
- name: Verify that PROMETHEUS is enabled
```

We don't need to mention the versions; it should work whenever the observability client is being used and prom is enabled. (technically, one doesn't need RHOSO18 for that